java/src/org/openqa/selenium/grid/node/local/LocalNode.java

```diff
@@ -176,8 +180,10 @@ private LocalNode(
       .ticker(ticker)
       .removalListener((RemovalListener<SessionId, TemporaryFilesystem>) notification -> {
         TemporaryFilesystem tempFS = notification.getValue();
-        tempFS.deleteTemporaryFiles();
-        tempFS.deleteBaseDir();
+        if (tempFS != null) {
+          tempFS.deleteTemporaryFiles();
+          tempFS.deleteBaseDir();
+        }
```

Comment:
Could you please revert this? It is not related to the PR.

Reply:
Fixed